config/public-api-v1.yml

```diff
@@ -31,31 +31,441 @@ paths:
+          description: "Are early career payments available for this course?"
+          example: true
+        has_scholarship:
+          type: boolean
+          description: "Are scholarships available for this course?"
+          example: true
+        has_vacancies:
+          type: boolean
+          description: "Do any of the locations for this course have vacancies?"
+          example: true
+        how_school_placements_work:
+          type: string
+          format: markdown
+          description: |
+            Additional information about the schools applicants will be teaching in.
+          example: how_school_placements_work
+        interview_process:
+          type: string
+          format: markdown
+          description: |
+            Additional information about how the interview process will work for applicants.
+          example: interview_process
+        is_send:
+          type: boolean
+          description: "Does this course have a SEND specialism?"
+          example: true
+        last_published_at:
+          type: string
+          format: date-time
+          description: |
+            Timestamp of when changes to this course's additional information
+            sections was published last.
+          example: "2019-06-13T10:44:31Z"
+        level:
+          type: string
+          description: "The level of pupils this course is designed for."
+          example: secondary
+          enum:
+            - further_education
+            - primary
+            - secondary
+        name:
+          type: string
+          description: "Name of the course."
+          example: "Art and Design"
+        open_for_applications:
+          type: boolean
+          description: "Is the course currently open for applications?"
+          example: true
+        other_requirements:
+          type: string
+          format: markdown
+          description: |
+            Any non-academic qualifications or documents the applicant may need.
+          example: other_requirements
+        personal_qualities:
+          type: string
+          description: |
+            Any skills, motivation and experience the provider is looking in applicants.
+          example: personal_qualities
+        program_type:
+          type: string
+          description: program_type
+          example: scitt_programme
+          enum:
+            - higher_education_programme
+            - scitt_programme
+            - school_direct_training_programme
+            - school_direct_salaried_training_programme
+            - pg_teaching_apprenticeship
+        provider_code:
+          type: string
+          description: "Unique code for the provider of this course."
+          maxLength: 3
+          example: 6CL
+        qualifications:
+          type: string
+          description: |
+            The qualifications as an outcome of the course.
+          example: [ qts ]
+          enum:
+            - [ qts ]
```

Comment:
Should it not be the human friendlier version?

Reply:
What would that be?

This renders as:

```
 Possible values:

  • ["qts"]
  • ["pgce"]
  • ["pgde"]
  • ["qts", "pgce"]
  • ["qts", "pgde"]
```

Reply:
was thinking of 
```ruby
[8] pry(main)> x = Course.all.pluck( :qualification).uniq
=> ["qts", "pgce_with_qts", "pgce", "pgde", "pgde_with_qts"]

```